api/allIncidents/incidentsRouter.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const router = express.Router();
+const Incidents = require('./incidentsModel');
+const { checkIncidentExists } = require('../middleware');
+
+// TODO document shape of objects coming and going
+
+/**
+ * @swagger
+ * /:
+ *  GET:
+ *    Summary: Path returning all approved incidents in reverse chronological order
+ *    tags:
+ *      - incidents
+ *    produces:
+ *      - application/json
+ *    responses:
+ *      200:
+ *        description: Success ... returns an array of incident objects
+ *      500:
+ *        description: Server response error
+ */
+
+router.get('/', (req, res, next) => {
+  Incidents.getApprovedIncidents()
```

Comment:
Will this really be getting only approved incident's or would that be something filtered in the front end after pulling all of the incidents?

Reply:
That will be your model function querying the database for only approved incidents :)